app/src/main/java/com/geekskool/dhobi/Adapters/PurchaseAdapter.java

```diff
@@ -0,0 +1,51 @@
+package com.geekskool.dhobi.Adapters;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CursorAdapter;
+import android.widget.TextView;
+
+import com.geekskool.dhobi.R;
+
+/**
+ * Created by manisharana on 19/2/17.
+ */
+public class PurchaseAdapter extends CursorAdapter {
+
+
+
+    public PurchaseAdapter(Context context, Cursor c, int flags) {
+        super(context, c, flags);
+    }
+    
+    @Override
+    public View newView(Context context, Cursor cursor, ViewGroup parent) {
+        View view = LayoutInflater.from(context).inflate(R.layout.purchase_list_row,parent,false);
+        view.setTag(new PurchaseViewHolder(view));
+        return view;
+    }
+
+    @Override
+    public void bindView(View view, Context context, Cursor cursor) {
+        PurchaseViewHolder vH = (PurchaseViewHolder) view.getTag();
+//        vH.pName.setText(cursor.getString(MainActivity.COL_COURSE_NAME));
+//        vH.pDate.setText(cursor.getString(MainActivity.COL_COURSE_LOCATION));
+//        vH.pAmount.setText(cursor.getString(MainActivity.COL_COURSE_NAME));         // // TODO: 20/2/17
```

Comment:
Try to avoid leaving any commented code in a commit, if possible. If you do need to leave it in, mark it clearly with a comment explaining why it's there.